src/Controller/ArticleController.php

```diff
@@ -83,53 +103,41 @@ public function addArticle(Request $request)
             $this->saveEntity($article);
 
             $this->addSuccessFlash(sprintf('Successfully saved new article "%s"', $article->getName()));
-            return $this->redirectToRoute('list_articles');
+            return $this->redirectToRoute('admin_list_articles');
         }
 
         return ['form' => $form->createView()] ;
     }
 
     /**
-     * @Route(
-     * "/admin/article/article_edit",
-     * name="edit_article",
-     * methods={"POST"},
-     * requirements={
-     *  "_format": "json"
-     *  }
-     * )
+     * @Route("/admin/article/edit/{id}", name="edit_article", requirements={"id"="\d+"})
+     * @Template("/admin/article/edit.html.twig")
      */
-    public function editArticleTitle(Request $request, ArticleRepository $articleRepository)
+    public function editJoy(Request $request, Article $article)
```

Comment:
@daria-sieroshtan 
In hell there is a place for those who copy-paste and there they In hell there is a place for those who copy-paste and there they